drivers/gpio/Kconfig.sx1509b

```diff
@@ -0,0 +1,47 @@
+# Kconfig.pcal9535a - PCAL9535A GPIO configuration options
+#
+#
+# Copyright (c) 2018 Aapo Vienamo
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig GPIO_SX1509B
+	bool "SX1509B I2C GPIO chip"
+	depends on GPIO && I2C
+	default n
+	help
+	  Enable driver for SX1509B I2C GPIO chip.
+
+if GPIO_SX1509B
+
+config GPIO_SX1509B_INIT_PRIORITY
+	int
+	default 70
+	prompt "Init priority"
+	help
+	  Device driver initialization priority.
+
+config GPIO_SX1509B_DEV_NAME
+	string "SX1509B GPIO chip Device Name"
+	depends on GPIO_SX1509B
```

Comment:
you can remove this depends on, since you are already using a condition on that option line 16